libraries/ms-helper/src/thermistor.c

```diff
@@ -20,6 +21,9 @@ static const uint32_t s_resistance_lookup[] = {
   999300,   973800,
 };
 
+// Used for converting the lookup table index with corresponding temperatures
+#define THERMISTOR_DECICELSIUS_RANGE_MAX (SIZEOF_ARRAY(s_resistance_lookup) - 1)
```

Comment:
I'm pretty sure this isn't the maximum range in 0.1C units. You have 101 entries and a range of 0 to 100C.